packages/prosemirror-lwdita-backend/src/server.ts

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
```

Comment:
@marmoure Could you please provide a basic API test that we can extend later once we define the routes?

Reply:
@plutonik-a That would be time costly, as we have to decide on which framework needed for testing and how exactly should we test the API.
can we leave it until we finish the happy case MVP?

Reply:
We will add the test at a later point then and add the open task in our backlog.